src/pages/index.js

```diff
@@ -42,34 +55,70 @@ cardModal.setEventListeners();
 const previewImageModal = new ModalWithImage("#preview-image-modal");
 previewImageModal.setEventListeners();
 
+const avatarEditModal = new ModalWithForm(
+  "#avatar-edit-modal",
+  handleAvatarImageSubmit
+);
+avatarEditModal.setEventListeners();
+
+const deleteCardModal = new ModalWithConfirmation(
+  "#delete-card-modal",
+  handleDeleteCard
+);
+deleteCardModal.setEventListeners();
+
 // USER INFO
 
 const userInfo = new UserInfo({
   nameElementSelector: ".profile__title",
-  jobElementSelector: ".profile__description",
+  aboutElementSelector: ".profile__description",
+  avatarImageSelector: ".profile__avatar",
+});
+
+document.addEventListener("DOMContentLoaded", () => {
+  api
+    .getUserInfoAndCards()
+    .then(([userData, initialCards]) => {
+      const avatarUrl = userData.avatar;
+      handleAvatarImageSubmit({ avatar: avatarUrl });
```

Comment:
`      handleAvatarImageSubmit({ avatar: avatarUrl });
`

You don't submit a form here. You just get the avatar link from the server and should set it. But everything is set with `userInfo.setUserInfo(userData)`